wechaty/user/message.go

```
@@ -147,6 +147,10 @@ func (m *Message) Date() time.Time {
 	return m.payload.Timestamp
 }
 
+func (m *Message) ReferMessage() *schemas.ReferMessagePayload {
+	return &(*m.payload.ReferMessage) // 不希望能被修改原数据
```

golangci-lint (1.64.8)

SA4001: &*x will be simplified to x. It will not copy x.

(staticcheck)